xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/http/Xs2aHttpLogSanitizer.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2018-2018 adorsys GmbH & Co KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.adorsys.xs2a.adapter.http;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class Xs2aHttpLogSanitizer {
+    private static final Logger logger = LoggerFactory.getLogger(Xs2aHttpLogSanitizer.class);
+    private static final String REPLACEMENT = "******";
+    private final List<String> sanitizedHeaders = new ArrayList<>();
+    private final List<Pattern> patterns = new ArrayList<>();
+    private JsonMapper objectMapper = new JsonMapper();
+
+    Xs2aHttpLogSanitizer() {
+        patterns.add(Pattern.compile("(consents|accounts)/[^/?\\s\\[\"]+(.*)"));
+        patterns.add(Pattern.compile("(authorisations)/[^/?\\s\\[\"]+(.*)"));
```

Comment:
Looks like this could be replaced by another or-clause in the regex above.

Reply:
fixed